<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="0" hostname="NAGSEZ545719L" failures="1" tests="1" name="edgeTest" time="15.75" errors="0" timestamp="2023-12-10T18:14:36 IST">
  <testcase classname="testClasses.TC_002_Forget_Password_Link" name="forget_password_test" time="4.035">
    <failure type="java.lang.AssertionError" message="null">
      <![CDATA[java.lang.AssertionError: null
at testClasses.TC_002_Forget_Password_Link.forget_password_test(TC_002_Forget_Password_Link.java:28)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)
at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1136)
at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635)
at java.base/java.lang.Thread.run(Thread.java:833)
... Removed 19 stack frames]]>
    </failure>
  </testcase> <!-- forget_password_test -->
</testsuite> <!-- edgeTest -->
